### PR TITLE
updated info about NDES server name

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2450

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 1]
```diff
@@ -425,7 +425,7 @@ Sign-in a workstation with access equivalent to a _domain user_.
 3. Under **MANAGE**, click **Application proxy**.
 4. Click **Configure an app**.
 5. Under **Basic Settings** next to **Name**, type **WHFB NDES 01**.  Choose a name that correlates this Azure AD Application Proxy setting with the on-premises NDES server.  Each NDES server must have its own Azure AD Application Proxy as two NDES servers cannot share the same internal URL.
-6. Next to **Internal Url**, type the internal fully qualified DNS name of the NDES server associated with this Azure AD Application Proxy.  For example, https://ndes.corp.mstepdemo.net).  This must match the internal DNS name of the NDES server and ensure you prefix the Url with **https**.
+6. Next to **Internal Url**, type the internal fully qualified DNS name of the NDES server associated with this Azure AD Application Proxy.  For example, https://ndes.corp.mstepdemo.net).  This must match the primary hostname (AD Computer Account name) of the NDES server and ensure you prefix the Url with **https**.
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md]:
```suggestion
6. Next to **Internal Url**, type the internal, fully qualified DNS name of the NDES server associated with this Azure AD Application Proxy.  For example, https://ndes.corp.mstepdemo.net).  You need to match the primary host name (AD Computer Account name) of the NDES server, and prefix the URL with **https**.
```

**Comment by e0i** [comment]
@mapalko Would you be able to review and merge this PR for a document that you have authored? The PR is based on issue #2450.

Thanks.

**Comment by mapalko** [comment]
I reviewed it and the change looks good.. I don't have merge rights. @DaniHalfin can you help merge this

**Comment by e0i** [comment]
@DaniHalfin Can you merge this PR that has passed review via _mapalko_ ?
Thanks.

**Pull request merged**